### PR TITLE
DOC: Improve setup for running the HelloWorld model in AzureML

### PR DESCRIPTION
**Pull request by ant0nsc**
Please follow the guidelines for PRs contained [here](docs/pull_requests.md). Checklist:

- [ ] Ensure that your PR is small, and implements one change.
- [ ] Add unit tests for all functions that you introduced or modified.
- [ ] Run PyCharm's code cleanup tools on your Python files.
- [ ] Link the correct GitHub issue for tracking.
- [ ] Update the [Changelog](CHANGELOG.md) file: Describe your change in terms of 
Added/Changed/Removed/... in the "Upcoming" section.
- [ ] When merging your PR, replace the default merge message with a description of your PR,
and if needed a motivation why that change was required.

### CODE REVIEW

File: docs/hello_world_model.md [hunk 1]
```diff
@@ -1,17 +1,77 @@
...
-    `Test/ML/test_data/train_and_test_data` and name the folder "hello_world"   
-    * If you have set up AzureML then parameter search can be performed for this model by running:
-    `python InnerEye/ML/runner.py --model=HelloWorld --azureml --hyperdrive`
+This model can be trained from the commandline from the root of the repo: `python InnerEye/ML/runner.py --model=HelloWorld`.
+When used like this, it will use dummy 3D scans as the training data, that are included in this repository. Training will run
+on your local dev machine.
+
+In order to get this model to train in AzureML, you need to upload the data to blob storage. This can be done via
+[Azure Storage Explorer](https://azure.microsoft.com/en-gb/features/storage-explorer/) or via the
+[Azure commandline tools](https://docs.microsoft.com/en-us/cli/azure/).
```
Comment (fepegar) [inline docs/hello_world_model.md:16]: Should we use a friendlier URL for this? E.g., https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-cli

**Review by fepegar — APPROVED** [review]
I just added a little suggestion for a URL, nothing critical.

I see you've installed a Markdown linter :D Maybe we should fix all the documents in an isolated PR, it'll be hard to see the changes otherwise.

File: docs/hello_world_model.md [hunk 2]
```diff
@@ -1,17 +1,77 @@
...
+
+Before uploading, you need to know what storage account you have set up to hold the data for your AzureML workspace, see
+[Step 4 in the Azure setup](setting_up_aml.md): For the upload you need to know the name of that storage account.
+
+## Option 1: Upload via Azure Storage explorer
+
+When starting Storage Explorer, you need to [log in to Azure](https://docs.microsoft.com/en-gb/azure/vs-azure-tools-storage-manage-with-storage-explorer?tabs=windows).
+
+* Select your subcription in the left-hand navigation, and then the storage account that you set up ealier.
```
Comment (mebristo) [inline docs/hello_world_model.md:25]: nit: typos in subscription and earlier

**Review by mebristo — APPROVED** [review]

File: docs/hello_world_model.md (hunk 1 above)
Comment (ant0nsc) [reply docs/hello_world_model.md:16]: I've clarified that detailed instructions follow below.

File: docs/hello_world_model.md [hunk 3]
```diff
@@ -13,16 +13,19 @@ on your local dev machine.
...
 [Step 4 in the Azure setup](setting_up_aml.md): For the upload you need to know the name of that storage account.
 
 ## Option 1: Upload via Azure Storage explorer
 
+First install [Azure Storage Explorer](https://azure.microsoft.com/en-gb/features/storage-explorer/).
+
 When starting Storage Explorer, you need to [log in to Azure](https://docs.microsoft.com/en-gb/azure/vs-azure-tools-storage-manage-with-storage-explorer?tabs=windows).
 
-* Select your subcription in the left-hand navigation, and then the storage account that you set up ealier.
+* Select your subcsription in the left-hand navigation, and then the storage account that you set up earlier.
```
Comment (fepegar) [inline docs/hello_world_model.md:28]: Not there yet :D

**Review by fepegar — CHANGES_REQUESTED** [review]

File: docs/hello_world_model.md [hunk 4]
```diff
@@ -1,17 +1,81 @@
...
+Before uploading, you need to know what storage account you have set up to hold the data for your AzureML workspace, see
+[Step 4 in the Azure setup](setting_up_aml.md): For the upload you need to know the name of that storage account.
+
+## Option 1: Upload via Azure Storage explorer
+
+First install [Azure Storage Explorer](https://azure.microsoft.com/en-gb/features/storage-explorer/).
+
+When starting Storage Explorer, you need to [log in to Azure](https://docs.microsoft.com/en-gb/azure/vs-azure-tools-storage-manage-with-storage-explorer?tabs=windows).
+
+* Select your subcsription in the left-hand navigation, and then the storage account that you set up earlier.
```
Comment (ant0nsc) [inline docs/hello_world_model.md:28]:
```suggestion
* Select your subscription in the left-hand navigation, and then the storage account that you set up earlier.
```

**Review by fepegar — APPROVED** [review]

**Pull request merged**